packages/squiggle-lang/__tests__/library/builtin_test.ts

```diff
@@ -67,7 +67,7 @@ describe("Operators", () => {
   });
 
   describe("try", () => {
-    testEvalToBe("try({|| 2+2}, {||0})", "4");
-    testEvalToBe("try({|| 2+''}, {||3})", "3");
```

Comment:
Previous code in this test now fails at the compile stage, so it's not possible to test `try` with it anymore.

Reply:
That's neat, early failures are definitely preferred.